scripts/nightly/upload.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+import pathlib
+import sys
+import json
+
+import boto3
+
+dist_folder = pathlib.Path.cwd() / 'dist'
+
+try:
+    f = next(dist_folder.glob('*.whl'))
+except StopIteration:
+    print("No .whl files found in ./dist!")
+    sys.exit()
+
+print("Uploading", f.name)
+s3 = boto3.client('s3')
+s3.upload_file(str(f), 'releases.wagtail.io', 'nightly/' + f.name, ExtraArgs={'ACL':'public-read'})
+
+print("Updating latest.json")
+
+boto3.resource('s3').Object('releases.wagtail.io', 'nightly/latest.json').put(Body=json.dumps({
+    "url": 'https://releases.wagtail.io/nightly/' + f.name,
```

Comment:
What do you think about replacing it with a `txt` containing just a link or a version of the nightly build? This way it will require less tools to get the actual URL (no need to have `jq` or use `grep`, etc): everything you need is `curl` or `wget -O-`.

So it will be something like: 

```
# URL in latest.txt
pip install $(curl https://releases.wagtail.io/nightly/latest.txt)

# Version in latest.txt
pip install https://releases.wagtail.io/nightly/package-name-$(curl https://releases.wagtail.io/nightly/latest.txt).whl

```

Reply:
I wasn't really thinking people would use this file directly. My plan is to make a static HTML page that retrieves this file and generates the correct pip install command.